pkg/network/ovn_kubernetes_test.go

```diff
@@ -2204,7 +2231,7 @@ func TestRenderOVNKubernetesEnableIPsec(t *testing.T) {
 		t.Errorf("The MachineConfig %s must exist, but it's not available", workerMachineConfigIPsecExtName)
 	}
 
-	// At the 2nd pass, test render logic while MC extension rollout is in progress.
+	// At the 3rd pass, test render logic while MC extension rollout is in progress.
```

Comment:
I am not sure we need the 5th pass. Wouldn't MCO be progressing here on 3rd pass? So `MachineConfigClusterOperatorReady` would be false?

Reply:
yes, i think so. removed 5th pass and checking it in 3rd pass itself.